go.mod

```diff
@@ -180,8 +170,8 @@ replace (
 	github.com/go-logr/zapr => github.com/go-logr/zapr v0.4.0
 	github.com/manifestival/manifestival => github.com/manifestival/manifestival v0.7.0
 	k8s.io/klog/v2 => k8s.io/klog/v2 v2.8.0
-
-	// Needed by metadata webhook and other components
-	knative.dev/networking => knative.dev/networking v0.0.0-20211101215640-8c71a2708e7d
-	knative.dev/pkg => knative.dev/pkg v0.0.0-20211210132429-e86584fd3c69
 )
+
+replace knative.dev/hack => knative.dev/hack v0.0.0-20220118141833-9b2ed8471e30
```

Comment:
That might help let's see!

Reply:
Yes, this is done automatically and is based on the actual version on the branch
https://github.com/openshift-knative/serverless-operator/pull/1515/files#diff-ff3402c713d8a504795b7115caa1af088b1bfc583d43adb5d0bc5a83aaa03bc1R34-R35